css/techfolio-theme/default.css

```
@@ -209,8 +209,23 @@ li[id]:hover .heading-anchor,
   color: var(--bs-link-color);
 }
 
+.heading-anchor svg {
+  width: 1.25em;
+  height: 1.25em;
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)